server/test/test.js

```diff
@@ -0,0 +1,195 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../app';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+describe('Fast-Food-App dummy data endpoint tests', () => {
+  describe('tests for valid inputs of Fast Food Fast API', () => {
+    it('should return code 201 with object of order just added', (done) => {
+      chai.request(app)
+        .post('/api/v1/orders')
+        .send({
+          name: 'Tracey Ezezubilike',
+          order: 'Spaghetti and Bottled water',
+          address: '6 NY street Ikoyi, Lagos',
+          status: 'wait',
+        })
+        .end((err, res) => {
+          expect(res).to.have.status(201);
+          expect(res.body).to.have.property('status');
+          expect(res.body.status).to.eql('success');
+        });
+
+      /**
+       * This is for seeding purposes for testing the get an order endpoint and get all orders
+       */
+      chai.request(app)
```

Comment:
seeding data is typically not done in a test case. Test cases are only supposed to hold data for that given test. Please look into this.